### DISCUSSION TITLE
Accessing Results of Middleware function?

### DISCUSSION THREAD

**Discussion by osseonews** [honojs/hono#2084] [Q&A]
Sorry for what might be a silly question, but how do you actually access the results of a middleware function in the next, do you just use c.set to set a variable and then c.get in the next function to access it? I don't see anything in the docs about this. Thanks.

**Comment by EdamAme-x** [comment]
right in saying that this is a case of getting results from other middleware from other middleware?

**Comment by yusukebe** [comment]
@osseonews 

> do you just use c.set to set a variable and then c.get in the next function to access it?

Yeah, exactly. Please use `c.set`/`c.get`.
